src/client/input/sdl.c

```diff
@@ -1373,6 +1508,33 @@ IN_Controller_Init(qboolean notify_user)
 				IN_Haptic_Effects_Info();
 			}
 
+#if SDL_VERSION_ATLEAST(2, 0, 16)	// support for controller sensors
+
+			if ( SDL_GameControllerHasSensor(controller, SDL_SENSOR_GYRO)
+				&& !SDL_GameControllerSetSensorEnabled(controller, SDL_SENSOR_GYRO, SDL_TRUE) )
+			{
+				float gyro_data_rate = SDL_GameControllerGetSensorDataRate(controller, SDL_SENSOR_GYRO);
+#ifndef _WIN32
```

Comment:
Could you write some comment about differences between gyro in _WIN32 platform and others?

Reply:
I did in the commit, but you're right, it should be more visible, especially since it's a feature that should get homogeneous between platforms in the (near?) future of SDL2. I'll add them in the code.

Seizing the opportunity, I'll be a little bit more verbose here:
SDL2 provides different gyro readings in Windows, compared to Linux and Mac for the Switch Pro Controller. Meaning, the same movement (let's say a 90° turn left) provides bigger numbers in Linux and Mac than in Windows. I have no proof, but seeing that the DualShock 4 provides the same numbers across all plaforms, and they coincide with the Pro Controller on Windows, I'd say the numbers in Linux and Mac are wrong. I'll ask on the SDL forums once yquake2 master gets stable and this can be tested by other people.
As you can see in the code, if the gyro's data rate is 200 Hz or less in anything "not Windows", it's a Pro Controller, so the "normalizing factor" is smaller. This factor is something used to achieve a "natural scale" in the final output, meaning a 90° turn left on your controller means a 90° turn left in-game, with sensitivity 1.0; having 2.0 sensi means a 180°, 0.5 is 45°... you get the idea. Messing this "normalizing factor" will not break the game or its functionality but kills the usefulness of the sensitivity. That's all.
I'll try to explain this in a shorter manner in the code comments... if I can :)

Thank you for your time to review this.